react/features/base/config/reducer.js

```diff
@@ -126,8 +129,10 @@ function _setConfig(state, { config }) {
 
     const newState = _.merge(
         {},
-        config,
-        { error: undefined },
+        config, {
+            error: undefined,
+            locationURL: state.locationURL
```

Comment:
The location URL was introduced there to be able to report errors to the external API. Now, with sessions, I guess it's possible we don't need it in the store?

Or, alternatively, we could store the config in a `config` attribute to make it easy to add metadata around it.

Reply:
URL's instance is also used to distinguish between current and outdated load config requests, but that is before SET_CONFIG. Now I need it later for the session, but I need to check why exactly, then I guess we can think if there are alternative solutions.

Reply:
Now it's needed here: https://github.com/jitsi/jitsi-meet/pull/3229/commits/9feec0732cba3cc4b07c2c52a753d1cdd8185fee#diff-dd5009ee7833e1f79d58154a5c723615R246